spec/_attachments/ic.did

```diff
@@ -322,10 +322,17 @@ type schnorr_public_key_result = record {
     chain_code : blob;
 };
 
+type schnorr_aux = variant {
+    bip341: record {
+      merkle_root_hash: blob;
+   }
+};
+
 type sign_with_schnorr_args = record {
     message : blob;
     derivation_path : vec blob;
     key_id : record { algorithm : schnorr_algorithm; name : text };
+    aux: opt schnorr_aux;
```

Comment:
Is the name `aux` clear/descriptive enough?

Reply:
I'm open to alternatives

Reply:
From interface spec meeting: If we don't see any such field being used for EdDSA, and the typing anyway does not guarantee consistency, we could also just have the `merkle_root_hash: opt blob` here.

Reply:
Re `aux` the consideration here is in the future we wanted to support additional fields such as the context field of `Ed25519ctx` (see https://www.rfc-editor.org/rfc/rfc8032) this approach would make it easier to support since we could add a new member to the variant.

Reply:
Maybe write it out as `auxiliary_parameters` so that it's a bit clearer (for people that do not spend their free time reading CS theory papers)?